tchannel/testing/vcr/server.py

```diff
@@ -84,7 +84,7 @@ def __init__(self, cassette, unpatch):
         VCRProxy.VCRServiceError,
     ))
     @gen.coroutine
-    def send(self, request, response, channel):
+    def send(self, request, response):
```

Comment:
hmmm... so I guess we'll have to get the sync client and vcr working with the top-level `tchannel.TChannel` too